pyomo/core/base/param.py

```diff
@@ -188,22 +188,6 @@ def is_parameter_type(self):
         """
         return True
 
-    def is_variable_type(self):
-        """
-        Returns False because this is not a variable object.
-        """
-        return False
-
-    def is_expression_type(self):
```

Comment:
I find the name of this method confusing - this is a separate conversation, but now that we have a base class PyomoModelObject, we can have a discussion about the names and intent of these methods.